src/common/ipc.ts

```diff
@@ -17,7 +17,11 @@ export const ipc_getDesktopAppPath = makeEndpoint.main(
   <{ path: string; exists: boolean }>_,
 );
 
-export const ipc_deleteFolder = makeEndpoint.main("deleteFolder", <{ path: string }>_, <SuccessPayload>_);
+export const ipc_migrateDolphin = makeEndpoint.main(
+  "deleteFolder",
+  <{ migrateNetplay: string | null; migratePlayback: string | null; desktopAppPath: string | null }>_,
```

Comment:
Why is the old desktop app path passed in as a parameter? Don't we already know what the old desktop app path is?

Reply:
atm the main process just generates it and the renderer stores it. I could have the main process regenerate it or store it, but i'm not sure where would be a good place to store it.

Reply:
Why does it need to be stored? Why can't it just get returned to itself, the same way it gets sent to renderer?